napari/_app_model/_submenus.py

```diff
@@ -30,4 +30,57 @@
         MenuId.MENUBAR_VIEW,
         SubmenuItem(submenu=MenuId.VIEW_SCALEBAR, title=trans._('Scale Bar')),
     ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_ACQUISITION, title=trans._('Acquisition')
+        ),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(submenu=MenuId.TOOLS_FILTERS, title=trans._('Filters')),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_TRANSFORM, title=trans._('Transform')
+        ),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_MEASUREMENT, title=trans._('Measurement')
+        ),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_CLASSIFICATION,
+            title=trans._('Classification'),
+        ),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_PROJECTION, title=trans._('Projection')
+        ),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_SEGMENTATION, title=trans._('Segmentation')
+        ),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_VISUALIZATION, title=trans._('Visualization')
+        ),
+    ),
+    (
+        MenuId.MENUBAR_TOOLS,
+        SubmenuItem(
+            submenu=MenuId.TOOLS_UTILITIES, title=trans._('Utilities')
+        ),
+    ),
 ]
```

Comment:
maybe this also could be autogenerated. I strongly prefer to have a single place to edit not accidentally to introduce cone inconsistency.

Reply:
The submenu defines hierarchy of the menu structure and is therefore business logic and require manual management. It can probably be somewhat autogenerated based on the id structure but afaik we don't have a fixed convention, and even if we do, I suspect we will have some outliers later

Reply:
maybe also this could be generated base on classmethod?

Reply:
Good call! I moved this as well to a classmethod